dataRetriever/resolvers/epochproviders/arithmeticEpochProvider.go

```diff
@@ -98,10 +95,12 @@ func (aep *arithmeticEpochProvider) computeCurrentEpoch() {
 		return
 	}
 
+	currentChainParameters := aep.chainParamsHandler.CurrentChainParameters()
+
 	diffTimeStampInSeconds := currentTimeStamp - aep.headerTimestampForNewEpoch
 	diffTimeStampInMilliseconds := diffTimeStampInSeconds * millisecondsInOneSecond
```

Comment:
these have to be updated as well for this to work, but it will came in a later PR